src/Commands/RunRepeater.spec.ts

```diff
@@ -0,0 +1,85 @@
+import 'reflect-metadata';
+import { RunRepeater } from './RunRepeater';
+import { type Cert } from '../RequestExecutor/Request';
+import { CertificatesLoader } from '../RequestExecutor';
+import https from 'node:https';
```

Comment:
Using module as a namespace goes against best practices. Ditto elsewhere

```suggestion
import { createServer, type Server } from 'node:https';
```